packages/@aws-cdk/aws-appmesh/lib/http-route-path-match.ts

```diff
@@ -83,10 +79,6 @@ class HttpRoutePrefixPathMatch extends HttpRoutePathMatch {
 class HttpRouteWholePathMatch extends HttpRoutePathMatch {
   constructor(private readonly match: CfnRoute.HttpPathMatchProperty) {
     super();
-
-    if (this.match.exact && this.match.exact[0] !== '/') {
-      throw new Error(`Exact Path for the match must start with \'/\', got: ${this.match.exact}`);
-    }
```

Comment:
Let's leave this validation unchanged.

Reply:
Well noted. 
We will be leaving the validation!